web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -57,6 +57,12 @@
 
 	let isNextButtonDisabled = false;
 
+	/** @type {number} */
+	let gasPrice;
```

Comment:
Shouldn't `gasPrice` and `gasLimit` be initialized with the received `gasSettings`?

e.g.
```js
let { gasLimit, gasPrice } = gasSettings;
```

Reply:
Depends, they do not need to be assigned the values from the settings store as the values get dispatched from the `GasControls` component.

Reply:
My whole point is that they shouldn't be dispatched on mount and initialized here.